packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/README.md

```diff
@@ -0,0 +1,28 @@
+# {{#titleCase}}{{project_name}}{{/titleCase}}
+
+## Project structure
+
+| Path | Contains |
+| ------------ | ------------ |
+| `lib/base/` | Common code used on more than one **feature** in the project. |
+| `lib/base/common_blocs/` | Global [BLoCs](https://pub.dev/packages/rx_bloc#business-logic-layer---bloc) |
+| `lib/base/common_ui_components/` | Reusable widgets (buttons, controls etc) |
+| `lib/base/common_use_cases/` | Global [UseCases](https://pub.dev/packages/rx_bloc#shared-business-logic-layer---service) |
+| `lib/base/extensions/` | Global [extension methods](https://dart.dev/guides/language/extension-methods) |
+| `lib/base/models/` | Data models used in the project |
+| `lib/base/remote_data_sources/` | External data sources like APIs. Here is placed all [retrofit](https://pub.dev/packages/retrofit) code. |
+| `lib/base/local_data_sources/` | Local data sources, such as shared preferences, secured storage etc. |
+| `lib/base/repositories/` | [Repositories](https://pub.dev/packages/rx_bloc#data-layer---repository) used to fetch and persist models.
```

Comment:
@DDavidPrime please update the links. also, take any useful information/snippets from [here](https://pub.dev/packages/rx_bloc)